pandasql/core.py

```diff
@@ -353,8 +357,9 @@ def __init__(self, data=None, name=None, sources=None, deep_copy=False):
 
         if df is not None and len(df) > 0:
             # Offload dataframe to SQLite
-            df.to_sql(name=self.name, con=SQL_CON, index=False)
+            df.to_sql(name=self.name, con=SQL_CON, index=False, chunksize=10000)
             self._cached_on_sqlite = True
+            self._input_size = self._cached_result.memory_usage(deep=True).sum()
```

Comment:
We should update `self._input_size` when the result is computed too, e.g., at the end of `compute`

Reply:
I'm not 100% sure about that. I guess it depends on how we are using `input_size`. Right now if we updated it after `compute`, the current cost model would count both the original `input_size` and the new `input_size` from the cached result.